languages/csharp/exercises/concept/chars/.docs/introduction.md

```diff
@@ -0,0 +1,10 @@
+The C# `char` type is a 16 bit quantity to represent the smallest addressable components of text.
```

Comment:
Nice and concise! The only thing that we need to add here is a section on the `StringBuilder`.